enclave/tls_cert.c

```diff
@@ -0,0 +1,164 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// clang-format off
+ #include <openenclave/bits/defs.h>
```

Comment:
remove initial whitespace in 5-6, for consistency with subsequent lines

Reply:
removed